filter/etc/e-smith/events/actions/nethserver-mail-filter-disable_rspamd

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+# 
+# This script is part of NethServer.
+# 
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+# 
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+use strict;
+
+use esmith::ConfigDB;
+my $cdb = esmith::ConfigDB->open() || die("Could not open ConfigDB");
+
+my $spamStatus = $cdb->get_prop('rspamd', 'SpamCheckStatus') || 'enabled';
+my $virusStatus = $cdb->get_prop('rspamd', 'VirusCheckStatus') || 'enabled';
+
+if (($spamStatus eq 'disabled') && ($virusStatus eq 'disabled'))
+    {
+        $cdb->set_prop('rspamd', 'status', 'disabled');
```

Comment:
Changing the DB values in an action is not recommended. The admin or the Server Manager has to set the prop, then run signal-event.